src/components/right_page/Gallery.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import deptEvent from "../../assets/img/dept_event.png";
+import deptMkt from "../../assets/img/dept_mkt.png";
+import deptPr from "../../assets/img/dept_pr.png";
+import deptTech from "../../assets/img/dept_tech.png";
+
+//padding and width for image should be adjusted when replacing image source
+export default function Gallery(props){
+    return(
+        <div className="gallery-container">
+            <div className="row">
+                <div className="col-12 col-md-6 col-lg-4">
+                    <div className="text-us">
+                        <h1 style={{"font-size": "3.5rem"}}>我們</h1>
+                    </div>
+                    <div className="text-line-middle"></div>
+                    <img src={deptMkt} alt="Failed to load" style={{"width": "75%", "float": "right"}}/>
+                    <img src={deptTech} alt="Failed to load" style={{"width": "80%", "float": "right"}}/>
```

Comment:
In fact, I suggest you add a `div` container for each image since in the future we might add features like "hover and show the text" or "click to zoom".